wordpress-mobile/MediaPicker-iOS
# Bumps pod version to 0.23

This PR updates the pod spec and documentation for release 0.23

To test:
 - Check if README and CHANGELOG are correct.
 - Check if pod specs files are correct.
